digitalocean/provider.go

```diff
@@ -67,6 +67,30 @@ func Provider() *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("SPACES_SECRET_ACCESS_KEY", nil),
 				Description: "The secret access key for Spaces API operations.",
 			},
+			"requests_per_second": {
+				Type:        schema.TypeFloat,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("DIGITALOCEAN_REQUESTS_PER_SECOND", 0.0),
+				Description: "The rate of requests per second to limit the HTTP client.",
+			},
+			"http_retry_max": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("DIGITALOCEAN_HTTP_RETRY_MAX", 0),
+				Description: "The maximum number of retries on a failed API request.",
+			},
+			"http_retry_wait_min": {
+				Type:        schema.TypeFloat,
```

Comment:
How about an `int` for this and `http_retry_wait_max`? Unless a float is necessary, they are a bit more user friendly.

Reply:
Floats aren't necessary, I thought about using them so it is possible to control at the millisecond level. Shall I update it to an `int` amount of seconds or keep it as a `float`? It is also possible to do things like string parsing (ex.: `1s200ms`) or receiving the duration in milliseconds (which I felt like it could be error prone for inattentive users).